test/functional/sqlsrv/sqlsrv_encrypted_query_nosqltype.phpt

```diff
@@ -23,26 +39,49 @@ Test using sqlsrv_query for binding parameters with column encryption and a cust
 
     function selectData()
     {
-        global $conn, $tableName;
+        global $conn, $tableName, $numRows, $patient;
 
         $stmt = sqlsrv_query($conn, "SELECT * FROM $tableName");
 
+        $id = 1;
         while ($obj = sqlsrv_fetch_object($stmt)) {
-            echo $obj->PatientId . "\n";
-            echo $obj->SSN . "\n";
-            echo $obj->FirstName . "\n";
-            echo $obj->LastName . "\n";
-            echo $obj->BirthDate . "\n\n";
+            if ($obj->PatientId !== $id) {
+                echo "Expected $id but got $obj->PatientId\n";
+            }
+            if ($obj->SSN !== $patient->SSN) {
+                echo "Expected $patient->SSN but got $obj->SSN\n";
+            }
+            if ($obj->FirstName !== $patient->FirstName) {
+                echo "Expected $patient->FirstName but got $obj->FirstName\n";
+            }
+            if ($obj->LastName !== $patient->LastName) {
+                echo "Expected $patient->LastName but got $obj->LastName\n";
+            }
+            if ($obj->BirthDate !== $patient->BirthDate) {
+                echo "Expected $patient->BirthDate but got $obj->BirthDate\n";
+            }
+            
+            $id++;
+        }
```

Comment:
Why do you have to compare the results in the test and not in expect? do ae with not ae return different results?

Reply:
yes, with AE, the error is specifically about specifying SQL type for each parameter, whereas without AE, there shouldn't be errors.

Reply:
ohhhhhh I see now after reading the test 3 times. So originally the test only works for Always Encrypted, and only one row is insert correctly. But now that it works for not Always Encrypted, all three attempts to insert rows are successful so there are 3 rows, with id = 1, 2 and 3, that's why you have to compare them instead of printing them out.

Reply:
thank you for your thorough review of this test :)